InternalDocs/garbage_collector.md

```diff
@@ -16,27 +16,27 @@ count field can be examined using the `sys.getrefcount()` function (notice that
 value returned by this function is always 1 more as the function also has a reference
 to the object when called):
 
-```pycon
-    >>> x = object()
-    >>> sys.getrefcount(x)
-    2
-    >>> y = x
-    >>> sys.getrefcount(x)
-    3
-    >>> del y
-    >>> sys.getrefcount(x)
-    2
+```python
```

Comment:
This is wrong.  It's actually an example of console session.  Please revert.

Reply:
Are you referring to the indentation change too, or just the **pycon** -> **python** change?

What's your opinion on the issue with line-comment highlighting that I mentioned in my previous comment? Non-issue?

Reply:
> Are you referring to the indentation change too

No, that part seems fine.

> What's your opinion on the issue with line-comment highlighting that I mentioned in my previous comment?

Seems to be a minor issue.

You might either prepend missing ">>> " or reformat some comments to be *after* code.  See e.g. the tutorial: https://docs.python.org/3/tutorial/introduction.html#lists